src/modules/vmware/application/dto/vmware-connection.dto.ts

```
@@ -0,0 +1,34 @@
+import { IsString, IsNumber, IsOptional, Min, Max } from 'class-validator';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import { Exclude } from 'class-transformer';
```

ESLint

[error] 'Exclude' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)